packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -158,7 +164,23 @@ class PaginatedDataTable extends StatefulWidget {
   ///
   /// This value is optional and defaults to kMinInteractiveDimension if not
   /// specified.
-  final double dataRowHeight;
+  @Deprecated(
+    'This property is no longer used, please use dataRowMinHeight and dataRowMaxHeight instead. '
+    'This feature was deprecated after v3.5.0-10.0.pre.',
+  )
+  final double? dataRowHeight;
+
+  /// The minimum height of each row (excluding the row that contains column headings).
+  ///
+  /// This value is optional and defaults to kMinInteractiveDimension if not
```

Comment:
This should have breadcrumbs here and below

```suggestion
  /// This value is optional and defaults to [kMinInteractiveDimension] if not
```